index.html

```diff
@@ -47,21 +48,15 @@ <h4>Fuel Analysis</h4>
 <h4>Reactor Design and Optimization</h4>
 <ul>
 <li><a href="{{ site.url }}users/colvinem/">Emory Colvin</a> (<a href="https://github.com/colvinem" class="user-mention">@colvinem</a>)</li>
-<li><a href="{{ site.url }}users/keppenj/">Jackson Keppen</a> (<a href="https://github.com/keppenj" class="user-mention">@keppenj</a>)</li>
 <li><a href="{{ site.url }}users/lilleybe/">Ben Lilley</a> (<a href="https://github.com/lilleybe" class="user-mention">@lilleybe</a>)</li>
 <li><a href="{{ site.url }}users/stewryan/">Ryan Stewart</a> (<a href="https://github.com/ryanstwrt" class="user-mention">@ryanstwrt</a>)</li>
-<li><a href="{{ site.url }}users/hillszach/">Zachary Hills</a> (<a href="https://github.com/zachthills" class="user-mention">@zachthills</a>)</li>
-</ul>
-
-<h4>Uncertainty Quantification</h4>
-<ul>
-<li><a href="{{ site.url }}users/combsky/">Kyle Combs</a> (<a href="https://github.com/Yoshimitsu45" class="user-mention">@Yoshimitsu45</a>)</li>
 </ul>
 
 <h4>RTRP Alumni</h4>
 <ul>
 <li><a href="https://ne.oregonstate.edu/tony-alberti">Tony Alberti</a> (<a href="https://github.com/albeanth" class="user-mention">@albeanth</a>)</li>
-<li><a href="{{ site.url }}users/harterj/">Jackson Harter</a> (<a href="https://github.com/harterj" class="user-mention">@harterj</a>)</li>
```

Comment:
I know you and Jackson Harter have legendary beef, but you can't erase his existence haha. Did you mean to delete this? You may have moved it and I am just not seeing it.

Reply:
Don't let him forget that I hold the power now! :) haha I pushed a new version with him included still